### ISSUE TITLE
Change keyboard shortcut for strike-through

### ISSUE THREAD

**Issue by pat-githubber** [nextcloud/text#2134]
**Describe the bug**
On my Mac in Firefox, I open a markdown file or create a new one, highlight any text so that the floating popup toolbar with `Add Link` and `Link File` pops up, and press COMMAND+SHIFT+X, the entire interface becomes flush right and all symbols are mirrored. Pressing COMMAND+SHIFT+X reverse the action to normal. When I close the file, and open it again, I have to highlight any text first so that COMMAND+SHIFT+X causes this unexpected behavior.

**To Reproduce with a new text file**
1. Create a `New text document.md` 
2. Open the document
3. Write any text in the document for example `abc`
4. Highlight `abc`
5. Now, keep text highlighted or click on any white space
6. Press COMMAND+SHIFT+X
7. Interface becomes flush right and all symbols are mirrored
8. Press COMMAND+SHIFT+X again and interface goes back to normal
9. Only causes this unexpected behavior when file is freshly opened and any text is highlighted for the first time

**To Reproduce with an existing text file**
1. Open an existing Markdown file 
2. Highlight any text
3. Now, keep text highlighted or click on any white space
4. Press COMMAND+SHIFT+X
5. Interface becomes flush right and all symbols are mirrored
6. Press COMMAND+SHIFT+X again and interface goes back to normal
7. Only causes this unexpected behavior when file is freshly opened and any text is highlighted for the first time

**Expected behavior**
Nothing would happen after pressing COMMAND+SHIFT+X

**Screenshots**
Initial state
![image](https://user-images.githubusercontent.com/87087327/151171310-4f1c9938-765a-4b7e-af2e-62639473eebf.png)

Highlighting text
![image](https://user-images.githubusercontent.com/87087327/151171485-cb694284-31b2-4680-903a-789e53fff960.png)

After pressing COMMAND+SHIFT+X (see also Sidebar content, which flipped as well)
![image](https://user-images.githubusercontent.com/87087327/151171590-4a984d2e-b56f-47d6-900b-626728cef8e2.png)

**Client details:**
 - OS: macOS Monterey 12.1
 - Browser: Firefox 96.0.2 (64-bit)
 - MacBookAir (M1, 2020)

**Comment by max-nextcloud** [comment]
COMMAND+SHIFT+X is the keyboard shortcut for right to left languages (Hebrew, Arabic, ...) in firefox. It also works in this text entry for this comment - so nothing text specific.
Which behavior would you have expected from this shortcut?

Closing this issue as the behavior is desired the way it is (unless people more familiar with rtl languages can specify how it would be done better). If you were looking for some specific functionality and lack a keyboard shortcut for it please let us know what you were missing the shortcut for.

**Comment by pat-githubber** [comment]
Sorry, it’s a nooby knowledge gap 😅

**Comment by juliushaertl** [comment]
@max-nextcloud Maybe we should consider changing the strikethrough shortcut then, as that would be the suggestion to the user for tiptap 2 https://github.com/nextcloud/text/pull/1981#issuecomment-1031493686

**Comment by max-nextcloud** [comment]
> @max-nextcloud Maybe we should consider changing the strikethrough shortcut then, as that would be the suggestion to the user for tiptap 2 [#1981 (comment)](https://github.com/nextcloud/text/pull/1981#issuecomment-1031493686)

Good catch. I'd say we go without a keyboard shortcut... or maybe ctrl+alt+S as the icon has an S in it and ctrl+S is save and ctrl+shif+S is "save as" in a lot of programms and screenshot in ff

**Comment by juliushaertl** [comment]
Lets reopen that then to address this.

Ctrl+Alt-S sounds reasonable

Found https://defkey.com/search?irq=strikethrough which I find quite a useful collections but seems there is no common pattern at least for strikethrough

**Comment by juliushaertl** [comment]
Was fixed upstream and added with https://github.com/nextcloud/text/pull/4710
